### DISCUSSION TITLE
Shows 'Threats Found' in App Protection.

### DISCUSSION THREAD

**Discussion by mrtear** [dovecoteescapee/ByeDPIAndroid#149] [General]
_is this normal?_

![img.jpg](https://github.com/user-attachments/assets/62e7573a-3166-404d-938f-8a223f132e59)

**Comment by deftikatik** [comment]
> _is this normal?_
> 
> ![img.jpg](https://github.com/user-attachments/assets/62e7573a-3166-404d-938f-8a223f132e59)
> 
> 

Yes, this is normal

**Comment by PomalinTheCat** [comment]
im kinda necroposting but im curious why does it trigger?

**Reply by mrtear** [reply]
idk what caused this. app didn't show anything like that after installing. I think I was messing with these settings:
UI editor (especially desync, protocols)

ps. sorry for the late reply
